care/users/api/viewsets/users.py

```diff
@@ -155,6 +167,16 @@ def get_queryset(self):
 
     def get_object(self) -> User:
         try:
+            if self.request.method == "GET" and not self.kwargs.get("username"):
```

Comment:
I don't see any new features here, why is this rebuilt ?

Reply:
> I don't see any new features here, why is this rebuilt ?

For making the user's details available to others (DRYPermissions had limited who had access to it); logged in user would be be able to view user details even if they don't have edit permissions.